addons/account/models/account_move.py

```diff
@@ -340,6 +340,12 @@ def _sequence_year_range_monthly_regex(self):
         copy=False,
         domain=[('display_type', 'in', ('product', 'line_section', 'line_note'))],
     )
+    invoice_sent_status = fields.Selection(
+        selection=[('sent', "Sent"), ('not_sent', "Not sent")],
+        compute='_compute_is_sent',
```

Comment:
Convention for the compute it should generally be like this, you take the name of the field as the name of compute (except if the compute is present on multiple field) 
```suggestion
        compute='_compute_invoice_sent_status',
```